keanu-project/src/main/java/io/improbable/keanu/network/NetworkWriter.java

```diff
@@ -0,0 +1,26 @@
+package io.improbable.keanu.network;
+
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.bool.BoolVertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.intgr.IntegerVertex;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public interface NetworkWriter {
```

Comment:
NetworkSaver or methods -> `write(...)`

Reply:
Can we fix this up once Ieva's stuff is out?  She's implementing the same interface so it would be better to clean once her stuff is out?

Reply:
👍